configure

```diff
@@ -580,8 +580,8 @@ MAKEFLAGS=
 # Identity of this package.
 PACKAGE_NAME='cmor'
 PACKAGE_TARNAME='cmor'
-PACKAGE_VERSION='3.5.0'
-PACKAGE_STRING='cmor 3.5.0'
+PACKAGE_VERSION='3.6.0'
+PACKAGE_STRING='cmor 3.6.0'
 PACKAGE_BUGREPORT='doutriaux1@llnl.gov'
```

Comment:
Same here

Reply:
I have changed the email from @doutriaux1 to mine.